src/components/select/simple-select/simple-select.spec.tsx

```diff
@@ -366,10 +366,10 @@ describe("SimpleSelect", () => {
         });
 
         describe("and the focus triggered by mouseDown", () => {
-          it("then that prop should not be called", () => {
+          it("then that prop should not be called more than once", () => {
```

Comment:
Yes, I'm not sure why it passed in the first place. @edleeks87 and me sat down and had a look at this one together but the test should have been failing as far as we could tell because that wasn't the behaviour that was actually in place. So the test title was updated and the test itself now checks for it be called once